### ISSUE TITLE
Allow line breaks in messages

### ISSUE THREAD

**Issue by ClementSparrow** [ClementSparrow/Pattern-Script#42]
Currently, if you want to make a message that is left-aligned rather than centered, or paragraphs in a message (and thus, lines that should stop at the end of the paragraph), or blank lines, it becomes a nightmare to find where to add spaces in the message so that the automatic centering function breaks the lines where you want. And some things are impossible without adding visible characters (e.g., a line break in the first line).

So I propose we can place manual line breaks in messages. The simplest way to do this would be to have a character that represents a line breaks and can be escaped, but it may break existing games. Other ways may be safer, but need to deal with the parser and I don't want to.

**Comment by david-pfx** [comment]
Dealing with the parser is right up my alley. How about this:
```
message This is a multi-line message
+ of exactly 4 lines
+
+ of which the 3rd is blank.
```
I don't think this is going to break any existing games.

**Comment by ClementSparrow** [comment]
Yeah, that would work. The spaces after a `+` should NOT be discarded, though.

**Comment by david-pfx** [comment]
I thought the idea was that any text would be trimmed and centered. Why would you retain spaces and then centre it off centre?
